controllers/phone-numbers/fetchDeletedPurchasedPhoneNumbers.js

```diff
@@ -5,6 +5,13 @@ const { fetchDeletedPurchasedPhoneNumbersFunc } = require("../../services/purcha
 const fetchDeletedPurchasedPhoneNumbers = async (req, res, next) => {
     const user = req.user;
 	try {
+		if (user.role !== 'CRM__ADMIN' && user.role !== 'CRM_MANAGER') {
```

Comment:
This is redundant and bad practise. use a middleware. Already implemented with routes in notifications or auth